crypto/kdf/kdf_local.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright (c) 2018, Oracle and/or its affiliates.  All rights reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+int KDF_str2ctrl(void *ctx, int (*ctrl)(void *ctx, int type, int p1, void *p2),
+                 int cmd, const char *str);
+int KDF_hex2ctrl(void *ctx, int (*ctrl)(void *ctx, int type, int p1, void *p2),
+                 int cmd, const char *hex);
+int KDF_md2ctrl(void *ctx, int (*ctrl)(void *ctx, int type, int p1, void *p2),
+                int cmd, const char *md);
```

Comment:
We traditionally lowercase internal symbols...

Reply:
Done.